addons/addon-base/packages/services/lib/aws/aws-service.js

```diff
@@ -39,6 +39,10 @@ class AwsService extends Service {
       this._sdk = AWSXRay.captureAWS(require('aws-sdk'));
     }
 
+    this._sdk.config.update({
+      customUserAgent: this.settings.get('customUserAgent'),
+    });
```

Comment:
what is the behavior if this field is missing? And how does local API testing work? I am guessing we don't want the SDK to have this enabled when service is deployed locally?